contrib/kafka/config/kafka.yaml

```diff
@@ -212,3 +212,32 @@ spec:
         - name: kafka-channel-controller-config
           configMap:
             name: kafka-channel-controller-config
+
+---
+
```

Comment:
You can add a comment here with a TODO to remove this in 0.7 and create a bug to remove it in 0.7.
It will also help someone installing the yaml to understand why this is present with replicas = 0.